apps/pxweb2/src/app/app.module.scss

```diff
@@ -1,85 +1,105 @@
 @use '$ui/style-dictionary/dist/scss/fixed-variables.scss' as fixed;
 
-.breakpoints {
-  height: 50px;
-  width: 100%;
+// Not from Figma
+.navigationAndContentContainer {
+  display: flex;
+  flex-direction: row;
 }
 
-.main {
+.mainContainer {
   display: flex;
-  width: 100%;
-}
+  background: var(--px-color-surface-subtle);
+  height: calc(100vh - 88px);
```

Comment:
Test this on a smartphone. A couple of years ago when scrolling at a page the application header for the browser (where the URL to the website is) would appear in certain browsers. 

In other words, try scrolling up or down, check if application header appears.

**Why it matters:**
Earlier (maybe still) when the application header appears the view height of the page changes, as the application header takes up some of the view height.

Reply:
It might not be a problem at all, just wanted to mention it

Reply:
When I scroll up and down on my iPhone I can see that the the header can be hidden/shown as you mention. It is very easy to display the header again by scrolling again. Do not know if this is an issue...

Reply:
That's not the issue. The issue I was thinking about is that since the view height (vh) changes, items might jump around on the page. If that does not happen, then everything is good 👍 

I should have explained better. Was in another meeting so was a bit distracted, sorry 😄